modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1840,6 +1840,13 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     getContentEditable,
     getContentEditableParent,
 
+    /**
+     * Checks if the specified node is editable within the given context of it's parents.
```

Comment:
```suggestion
     * Checks if the specified node is editable within the given context of its parents.
```
See [my notes in Notion](https://notion.so/tinycloud/Notes-on-possessive-apostrophes-vs-contraction-apostrophes-5dfb8def1b054de28e4ad5d5cbe63631) for all the gory details behind this tiny edit.